tee-worker/local-setup/py/helpers.py

```diff
@@ -45,10 +45,6 @@ def setup_working_dir(source_dir: str, target_dir: str):
         else:
             print(f'{source} does not exist. Did you run make?')
 
-def copy_config_files(source_dir: str, target_dir: str):
-    files_to_copy = ['worker-config-dev.json', 'worker-config-mock.json', 'worker-config-prod.json', 'worker-config-staging.json']
-    [shutil.copy(f'{source_dir}/{f}', f'{target_dir}/{f}') for f in files_to_copy]
-
```

Comment:
nice catches. Thanks. It would be better to make a separate PR. But keep it as it is.

Reply:
This will be covered in https://github.com/litentry/litentry-parachain/pull/1310, but yea let's just leave it